app/assets/stylesheets/components/header.scss

```
@@ -124,22 +124,29 @@
 
 .skip-content-link {
   position: absolute;
-  top: 0;
   left: 50%;
-  transform: translate(-50%, -100px);
   background: var(--body-color);
   color: var(--body-color-inverted);
   z-index: var(--z-popover);
   padding: var(--su-4);
   pointer-events: none;
-  opacity: 0;
   border-radius: var(--radius);
-  outline: 0;
+  height: var(--header-height);
+  transform: translate(-50%, calc(-1 * var(--header-height)));
```

Why not this:

```suggestion
  transform: translate(-50%, -100%);
```

..since it's height is equal to `--header-height`

---

why did I not think of that 😄  - so much better! Thanks for these suggestions 👍